### PR TITLE
[ffmpeg] install correct copyright file and enable (L)GPLv3 builds

### PR DESCRIPTION
**Pull request by mcmtroffaes**
This patch adds a new "version3" feature to allow (L)GPLv3 builds. It also inspects the log files to ensure that the correct license file is installed.

### CODE REVIEW

**Comment by cbezault** [comment]
This is an awesome patch, thanks!

File: ports/ffmpeg/portfile.cmake [hunk 1]
```diff
@@ -257,8 +261,23 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR
...
+    set(LICENSE_FILE "COPYING.GPLv2")
+elseif(${LICENSE_STRING} STREQUAL "License: GPL version 3 or later")
+    set(LICENSE_FILE "COPYING.GPLv3")
+elseif(${LICENSE_STRING} STREQUAL "License: nonfree and unredistributable")
+    set(LICENSE_FILE "COPYING.NONFREE")
+    file(WRITE ${SOURCE_PATH}/${LICENSE_FILE} ${LICENSE_STRING})
+else()
+    message(FATAL_ERROR "Failed to identify license (${LICENSE_STRING})")
+endif()
+message(STATUS ${LICENSE_STRING})
```
Comment (JackBoosY) [inline ports/ffmpeg/portfile.cmake]: Can you remove this debug message print?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/ffmpeg/portfile.cmake (hunk 1 above)
Comment (mcmtroffaes) [reply ports/ffmpeg/portfile.cmake]: Sure, done.

**Review by JackBoosY — APPROVED** [review]

**Review by dan-shaw — APPROVED** [review]

**Comment by dan-shaw** [comment]
LGTM, waiting for MacOS pipelines to be back up before merging

**Pull request merged**